extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -1388,7 +1393,8 @@ export class AmpDatePicker extends AMP.BaseElement {
       props.orientation = 'verticalScrollable';
       props.withFullScreenPortal = true;
     }
-
+    
+    props.hideKeyboardShortcutsPanel = this.hideKeyboardShortcutsPanel_;
```

Comment:
You can add `'hide-keyboard-shortcuts-panel'` into the `attributesToForward` instead since it is a property whose value does not change in the lifetime of the element.